pennylane/data/attributes/operator/operator.py

```diff
@@ -125,7 +125,7 @@ def consumes_types(cls) -> FrozenSet[Type[Operator]]:
                 qml.SpecialUnitary,
                 # pennylane/ops/state_preparation.py
                 qml.BasisState,
-                qml.QubitStateVector,
```

Comment:
We should make sure that `qml.QubitStateVector` operations are still compatible with datasets. We may want to include both in this list for now.